### ISSUE TITLE
Tutorials for In-silico Reverse Perturbation Prediction

### ISSUE THREAD

**Issue by taffy-miao** [bowang-lab/scGPT#59]
Hi TEAM!
Your works are marvelous and inspired me a lot. But I have faced some problems:
I don't know how to repeat the Figure3:D and F result, I wonder if there's a unpublished tutorial for the _In-seilico-experinment_?
Thanks in advance!

**Comment by subercui** [comment]
Hi, thank you for your interest. We haven't released that one yet. Originally, we didn't plan to include that in the tutorials folder. We will make a separate notebook for it in the near future.

**Comment by nathaniel-mahieu** [comment]
I would like to echo appreciation for this publication/model!

And a tutorial for reverse perturbation prediction would be greatly appreciated!

**Comment by lautaro450** [comment]
Hello, is there any update regarding the tutorial notebook for In-silico reverse perturbation prediction?.

**Comment by linlull** [comment]
> Hello, is there any update regarding the tutorial notebook for In-silico reverse perturbation prediction?.

Yes, I found the tutorial notebook for In-silico reverse perturbation prediction in https://zenodo.org/records/10466117.
